packages/react/src/toolbar/button/ToolbarButton.test.tsx

```diff
@@ -380,7 +380,7 @@ describe('<Toolbar.Button />', () => {
           expect(trigger).toHaveFocus();
         });
 
-        expect(handleValueChange.callCount).to.equal(1);
+        expect(handleValueChange.callCount).to.equal(2);
```

Comment:
Should be fixed after https://github.com/mui/base-ui/pull/1597, changed it here for now